crates/fluvio-smartengine/src/engine/wasmtime_engine/engine.rs

```diff
@@ -77,27 +72,12 @@ impl SmartModuleChainBuilder {
         })
     }
 }
-
-impl<T: Into<Vec<u8>>> From<(SmartModuleConfig, T)> for SmartModuleChainBuilder {
-    fn from(pair: (SmartModuleConfig, T)) -> Self {
-        let mut result = Self::default();
-        result.add_smart_module(pair.0, pair.1.into());
-        result
-    }
-}
-
 /// SmartModule Chain Instance that can be executed
 pub struct SmartModuleChainInstance {
```

Comment:
Since this Wasmtime specific, this should be named `SmartModuleWasmTimeChainInstance` or something like that.

Reply:
1. This PR doesn't introduce wasm abstractions (The "imp" style), renaming it would break public APIs. Or would you like `pub use self::wasmtime:: SmartModuleWasmTimeChainInstance as SmartModuleChainInstance`?
2. It's already in `wasmtime` directory, so I think adding `Wasmtime` to the name is optional.

How do you think?